From 149b46ede5844e3c4645a08433e8f855403211c7 Mon Sep 17 00:00:00 2001
From: PowerAI CI Functional ID <pwraiciw@us.ibm.com>
Date: Mon, 13 Apr 2020 21:01:12 +0000
Subject: [PATCH] fix constant_op_test

Add missing input parameter to constant_op_test.cc

This line: `OpKernelContext ctx(&params);` core dumps constant_op_test,
every other places that line is used works. The difference is in each
case params includes params.inputs which in this input tensor.

Adding that fixes this test failure
---
 tensorflow/core/kernels/constant_op_test.cc | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tensorflow/core/kernels/constant_op_test.cc b/tensorflow/core/kernels/constant_op_test.cc
index 7f424b4999..01d233ab1f 100644
--- a/tensorflow/core/kernels/constant_op_test.cc
+++ b/tensorflow/core/kernels/constant_op_test.cc
@@ -66,11 +66,15 @@ void ConstantOpTest::PersistentMemoryTrackingTest(bool on_gpu) {
                                               TF_GRAPH_DEF_VERSION, &status));
   TF_ASSERT_OK(status);
 
+  gtl::InlinedVector<TensorValue, 4> inputs;
+  inputs.push_back({nullptr, &tensor});
+
   OpKernelContext::Params params;
   params.device = device.get();
   params.frame_iter = FrameAndIter(0, 0);
   params.op_kernel = op.get();
   params.track_allocations = true;
+  params.inputs = &inputs;
 
   OpKernelContext ctx(&params);
   op->Compute(&ctx);
-- 
2.23.0

